### PR TITLE
Node gyp fix

### PR DESCRIPTION
**Pull request by marek-szydlowski-wizzdev**

### CODE REVIEW

File: MicroPython/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 wheel
 esptool==2.8
+gitpython
```
Comment (damian-kurek-wizzdev) [inline MicroPython/requirements.txt:3]: missing version

File: terraform/README.md [hunk 2]
```diff
@@ -118,7 +151,8 @@ Next, you should activate it. This step is platform dependent:
 
 ### 2. Install requirements
 ```
-pip3 install -r requirements.txt
+pip install pyserial cryptography click future pyelftools setuptools
```
Comment (damian-kurek-wizzdev) [inline terraform/README.md:154]: Add line that venv needs to be activated

File: terraform/scripts/requirements.txt [hunk 3]
```diff
@@ -1,9 +1,7 @@
-boto3
-botocore
+boto3==1.12.10
 jmespath
 setuptools
```
Comment (damian-kurek-wizzdev) [inline terraform/scripts/requirements.txt:3]: add versions

**Review by damian-kurek-wizzdev — COMMENTED** [review]
looks fine I added comments for minor changes

File: terraform/README.md (hunk 2 above)
Comment (marek-szydlowski-wizzdev) [reply terraform/README.md:154]: Requirements installation follows the activation of virtual environments - there is information about it two lines above

File: terraform/scripts/requirements.txt (hunk 3 above)
Comment (marek-szydlowski-wizzdev) [reply terraform/scripts/requirements.txt:3]: Added version to jmespath and skipped version number to setuptools as the version depends on the version of Python (the minor version) and it is installed automatically

File: MicroPython/requirements.txt (hunk 1 above)
Comment (marek-szydlowski-wizzdev) [reply MicroPython/requirements.txt]: Added version to gitpython (skipped to wheel, as it is the same case as with setuptools - depending on minor version of Python)

**Review by damian-kurek-wizzdev — APPROVED** [review]

**Pull request merged**